resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/DepsTestsSuite.scala

```diff
@@ -371,7 +371,7 @@ private[spark] trait DepsTestsSuite { k8sSuite: KubernetesSuite =>
   }
 
   private def setPythonSparkConfProperties(conf: SparkAppConf): Unit = {
```

Comment:
Shall we remove this function? Seems like it's only used once, and doesn't look helping the readability much.

Reply:
make sense, addressed, also updated PR title and description